DayZExpansion/AI/Scripts/4_World/DayZExpansion_AI/Classes/Patrols/eAIDynamicPatrol.c

```diff
@@ -133,8 +133,18 @@ class eAIDynamicPatrol : eAIPatrol
 		pos = ExpansionAIPatrol.GetPlacementPosition(pos);
 
 		eAIBase ai;
-		if (!Class.CastTo(ai, GetGame().CreateObject(GetRandomAI(), pos))) return null;
+		if (!Class.CastTo(ai, CreateAIEntity(pos))) return null;
 
+		ConfigureAi(ai, pos);
+
+		return ai;
+	}
+
+	private Object CreateAIEntity(vector pos) {
+		return GetGame().CreateObject(GetRandomAI(), pos);
+	}
+
+	private void ConfigureAi(eAIBase ai, vector pos) {
 		ai.SetPosition(pos);
```

Comment:
Not sure why is this need cos we already passed position to `CreateObject` but kept it untouched.

Reply:
Options to set the specific AI class names used to spawn the AI patrol units have been added in the last update:
https://github.com/salutesh/DayZ-Expansion-Scripts/blob/263948278268d5cda105caa0d9145c23fc38a480/DayZExpansion/AI/Scripts/4_World/DayZExpansion_AI/Classes/Patrols/eAIDynamicPatrol.c#L141